src/config/getStatusBarHeight.js

```diff
@@ -0,0 +1,11 @@
+import { getStatusBarHeight as ovrGetStatusBarHeight } from 'react-native-status-bar-height';
```

Comment:
I am confused by the naming of `ovrGetStatusBarHeight`. what does `ovr` stand for in that?

Reply:
`react-native-status-bar-height` author @ovr )))